css/src/components/timeline.scss

```diff
@@ -0,0 +1,51 @@
+$timeline-content-font-size: $font-size-8 !default;
+$timeline-timestamp-font-size: $font-size-9 !default;
+$timeline-timestamp-font-weight: $weight-semilight !default;
+$timeline-description-padding-top: $spacer-4 !default;
+$timeline-description-line-height: $line-height-normal;
+
+$timeline-item-border-left: $border-width solid $border !default;
+$timeline-item-min-height: $layout-6 !default;
+$timeline-item-padding-bottom: $spacer-7 !default;
+$timeline-item-padding-left: $spacer-8 !default;
+
+$timeline-badge-top-offset: -0.35em !default;
+$timeline-badge-left-offset: -1.05em !default;
+
+.timeline-item {
```

Comment:
Might be a good time to consider a container query for this. We should scale down to 320 pixel wide without issue:
<img width="203" alt="image" src="https://github.com/user-attachments/assets/9c0dec89-1828-4c12-9e2e-f3741e9d725d">

Reply:
Based on local testing, `flex-wrap: wrap` might be sufficient for most cases? Maybe adding a `gap` as well.

Reply:
Fixed via conversation